metron-analytics/metron-profiler/src/main/java/org/apache/metron/profiler/bolt/ProfileHBaseMapper.java

```diff
@@ -0,0 +1,136 @@
+/*
+ *
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ *  or more contributor license agreements.  See the NOTICE file
+ *  distributed with this work for additional information
+ *  regarding copyright ownership.  The ASF licenses this file
+ *  to you under the Apache License, Version 2.0 (the
+ *  "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package org.apache.metron.profiler.bolt;
+
+import backtype.storm.tuple.Tuple;
+import org.apache.hadoop.hbase.util.Bytes;
+import org.apache.metron.profiler.ProfileMeasurement;
+import org.apache.storm.hbase.bolt.mapper.HBaseMapper;
+import org.apache.storm.hbase.common.ColumnList;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Calendar;
+
+/**
+ * An HbaseMapper that defines how a ProfileMeasurement is persisted within an HBase table.
+ */
+public class ProfileHBaseMapper implements HBaseMapper {
+
+  /**
+   * A salt is prepended to the row key to help prevent hotspotting.  This constant is used
+   * to generate the salt.  Ideally, this constant should be roughly equal to the number of
+   * nodes in the Hbase cluster.
+   */
+  private int saltDivisor = 10;
+
+  /**
+   * The name of the column family.
+   */
+  private String columnFamily;
+
+  public ProfileHBaseMapper() {
+    setColumnFamily("cfProfile");
```

Comment:
Generally people try to keep the CF name super short (often 1 character) because it's replicated with every data point (see http://hbase.apache.org/0.94/book/rowkey.design.html).  Can we call it 'P' for profile?